Exiled.API/Features/Respawn.cs

```diff
@@ -23,6 +25,16 @@ public static class Respawn
         /// </summary>
         public static SpawnableTeamType NextKnownTeam => RespawnManager.Singleton.NextKnownTeam;
 
+        /// <summary>
+        /// Gets the amount of seconds before the next respawn will occur.
+        /// </summary>
+        public static int TimeUntilRespawn => Mathf.RoundToInt(RespawnManager.Singleton._timeForNextSequence - (float)RespawnManager.Singleton._stopwatch.Elapsed.TotalSeconds);
+
+        /// <summary>
+        /// Gets a value indicating whether or not a team is currently being spawned or the animations are playing for a team.
+        /// </summary>
+        public static bool CurrentlySpawning => RespawnManager.Singleton._curSequence == RespawnManager.RespawnSequencePhase.PlayingEntryAnimations || RespawnManager.Singleton._curSequence == RespawnManager.RespawnSequencePhase.SpawningSelectedTeam;
```

Comment:
It'd better to rename the variable to `IsSpawning`

Reply:
True, I didn't think about this. I won't be at my pc for a bit if you'd like to change it.